ajax/libs/nicedit/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "nicedit",
+  "version": "0.93",
+  "description": "Lightweight, Cross Platform WYSIWYG editor.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/danishkhan/NicEdit.git"
+  },
+  "author": {
+    "name": "Danish Khan"
+  },
```

Comment:
Let's modify to `"author": "Danish Khan",`
Because there is only one line.
Thanks